app/endpoints.php

```diff
@@ -0,0 +1,498 @@
+<?php
+
+use Utopia\Database\Adapter;
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Query;
+use Utopia\Http\Http;
+use Utopia\Http\Response;
+use Utopia\Http\Validator\ArrayList;
+use Utopia\Http\Validator\Assoc;
+use Utopia\Http\Validator\Boolean;
+use Utopia\Http\Validator\FloatValidator;
+use Utopia\Http\Validator\Integer;
+use Utopia\Http\Validator\Text;
+
+Http::get('/mock/error')
+    ->groups(['api', 'mock'])
+    ->action(function () {
+        throw new Exception('Mock error', 500);
+    });
+
+Http::get('/v1/ping')
+    ->groups(['api'])
+    ->inject('adapter')
+    ->inject('response')
+    ->action(function (Adapter $adapter, Response $response) {
+        $output = $adapter->ping();
+
+        $response->json([
+            'output' => $output
```

Comment:
I put wrapping back again, its needed because some outputs arent JSON formatted, and it would break response structure in some scenarios - such as boolean response